src/redux/books/books.js

```diff
@@ -16,7 +18,28 @@ export const removeBook = (id) => ({
   },
 });
 
-const initialState = [];
+const initialState = [
+  {
+    id: uuidv4(),
+    title: 'A Song of Ice and Fire',
+    author: 'Ramin Djawadi',
+  },
+  {
+    id: uuidv4(),
+    title: 'Mere Living',
+    author: 'C.S Lewis',
+  },
+  {
+    id: uuidv4(),
+    title: 'Never Split the Difference',
+    author: 'Chris Voss',
+  },
+  {
+    id: uuidv4(),
+    title: 'Getting to Yes',
+    author: 'William Ur',
+  },
+];
 
```

Comment:
- [ ] Great job.. Everything works ok. please remove the `Gothic` span, as it adds to all new books created. 
this screenshot is due to the fact that i can't see your code to comment under. Please don't always merge when your code is yet to be reviewed.
![Screenshot (316)_LI](https://user-images.githubusercontent.com/65298048/186200413-76b6797f-1105-4337-9929-4a33a968812f.jpg)

Reply:
oooppsss ok my bad then .. Please resubmit, so the next reviewer can approve… Great job and happy coding 😄

Reply:
Hello @tufoinnkuo10 . Thanks for responding. I've updated the book.